controllers/storagecluster/reconcile.go

```diff
@@ -49,6 +49,7 @@ mon_osd_backfillfull_ratio = .8
 mon_osd_nearfull_ratio = .75
 [osd]
 osd_memory_target_cgroup_limit_ratio = 0.5
+mon_max_pg_per_osd = 300
```

Comment:
This needs to move up to the `[global]` section to apply to the mons instead of the `[osd]` section. Confusing, I agree. :)

Reply:
Addressed!